relay/implementations/beacon-api.ts

```diff
@@ -631,6 +631,15 @@ export class BeaconApi implements IBeaconApi {
       console.log(this.getCurrentApi());
       try {
         const result = await this.fetchWithFallbackNoRetry(subUrl, init);
+
+        if (result.status == 404) {
+          logger.warn('404 not found');
+          logger.warn('Retrying as sometimes info appears with lag');
+          await sleep(1000);
+          this.nextApi();
+          continue;
+        }
```

Comment:
I don't want to sleep for too long :smiley:

Reply:
Depending on how much the delay is, you might falsely assume you won't get it

Reply:
It works like this I think the problem is in the API because we are requesting data for the current head slot returned by the same API. We can't handle a case where it becomes available after 2 hours